official/guides/child_account_update/export.py

```diff
@@ -0,0 +1,154 @@
+parser.add_argument("-k", "--key", required=True, help="Parent production API key")
+
+
+def authenticate(key: str):
+    """
+    Authenticate with the given API key
+
+    :param key: API key to authenticate with
+    :type key: str
+    :raises ValueError: If the key is invalid
+    :return: None
+    """
+    if not key:
+        raise ValueError("API key not provided")
+    easypost.api_key = key
+
+
+def get_production_key(keys: List) -> Union[str, None]:
+    """
+    Get the production key from a list of keys
+
+    :param keys: List of keys
+    :type keys: List
+    :return: Production key if found, None otherwise
+    :rtype: str | None
+    """
+    for key in keys:
+        if key["mode"] == "production":
+            return key["key"]
+    return None
+
+
+def process_child(child: easypost.User) -> Dict:
+    """
+    Process a child account
+
+    :param child: Child account object
+    :type child: easypost.User
+    :return: Dictionary of child account information
+    :rtype: Dict
+    """
+    child_prod_key = get_production_key(child.keys)
+    authenticate(child_prod_key)
+    accounts = get_usps_accounts()
+    credentials = []
+    for account in accounts:
+        account_details = {
+            "test": account.test_credentials,
+            "prod": account.credentials,
+        }
+        if account_details.get("test"):
+            details = {**account_details.get("test"), "carrier_account_id": account.id, "credential_type": "test"}
+            credentials.append(details)
+        if account_details.get("prod"):
+            details = {**account_details.get("prod"), "carrier_account_id": account.id, "credential_type": "prod"}
+            credentials.append(details)
+
+    return {"id": child.id, "accounts": credentials}
+
+
+def get_usps_accounts() -> List[easypost.CarrierAccount]:
+    """
+    Get all USPS accounts for the current user
+
+    :return: List of USPS accounts
+    :rtype: List[easypost.CarrierAccount]
+    """
+    accounts = easypost.CarrierAccount.all()
+    usps_accounts = []
+    for account in accounts:
+        if account.type == "UspsAccount":
+            usps_accounts.append(account)
+    return usps_accounts
+
+
+def write_to_csv(data: List[Dict]):
+    """
+    Write the data to a CSV file
+
+    :param data: Child account data
+    :type data: List[Dict]
+    :return: None
+    """
+    with open(OUTPUT_FILE_NAME, "w") as csvfile:
+        fieldnames = [
+            "child_id",
+            "carrier_account_id",
+            "credential_type",
+            "city",
+            "state",
+            "street",
+            "zip",
+            "company",
+            "email",
+            "phone",
+            "shipper_id",
+        ]
+        writer = csv.DictWriter(csvfile, fieldnames=fieldnames)
+        writer.writeheader()
+        rows = []
+        for child in data:
+            for credential in child["accounts"]:
```

Comment:
`account` is probably a better word here since user's could have multiple USPS accounts

```suggestion
            for account in child["usps_accounts"]:
```

Reply:
Resolved